build.cake

```diff
@@ -36,6 +39,26 @@ Action<FilePath, DirectoryPath, ProcessArgumentBuilder> Cmd => (path, workingPat
     }
 };
 
+///////////////////////////////////////////////////////////////////////////////
+// GLOBAL VARIABLES
+///////////////////////////////////////////////////////////////////////////////
+
+if (!Kudu.IsRunningOnKudu)
+{
+    throw new Exception("Not running on Kudu");
+}
+
+var deploymentPath = Kudu.Deployment.Target;
+if (!DirectoryExists(deploymentPath))
```

Comment:
This will fail if build on any other environment than kudu, so this should probably be in the "Kudu-Sync" task.